src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/ttc/Avenir.ttc");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-block {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin: 20px 0;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;

```

Comment:
ok, but better http://codeguide.academy/html-css.html#css-order

https://medium.com/swlh/better-ways-to-organise-css-properties-9a066e7ded62